src/celeritas/optical/CherenkovGenerator.hh

```diff
@@ -173,7 +171,7 @@ CELER_FUNCTION TrackInitializer CherenkovGenerator::operator()(Generator& rng)
         = rotate(from_spherical(-std::sqrt(sin_theta_sq), phi), dir_);
 
     // Sample fraction along the step
-    UniformRealDistribution sample_step_fraction;
+    UniformRealDistribution<> sample_step_fraction;
```

Comment:
Do you know why this is now required?

Reply:
This caused nvcc to segfault: https://github.com/celeritas-project/celeritas/pull/1787/commits/d2c4020cf8af1ef0beb9def9d59c6eb582cd1651 some sort of bug.